resources/js/photoinit.js

```diff
@@ -196,6 +209,32 @@ function initPhotoSwipeFromDOM (gallerySelector) {
         photoBooth.toggleMailDialog(img);
     });
 
+    /* slideshow management */
+    $(ssButtonClass).on('click touchstart', function(e) {
+        e.preventDefault();
+        e.stopPropagation();
+        // toggle slideshow on/off
+        setSlideshowState(this, !ssRunning);
+    });
+
+    function setSlideshowState(el, running) {
+        if (running) {
+            setTimeout(gotoNextSlide, ssDelay / 2.0);
+        }
+        const title = running ? 'Pause Slideshow' : 'Play Slideshow';
+        $(el).removeClass(running ? 'play' : 'pause')
+            .addClass(running ? 'pause' : 'play')
+            .prop('title', title);
+        ssRunning = running;
+    }
+
+    function gotoNextSlide() {
+        if (ssRunning && Boolean(gallery)) {

```

Comment:
Why is the `Boolean` constructor used at this point?

Reply:
`yarn eslint` told so....

Reply:
Never saw this before and I don't get the advantage, but it's also no big deal.